.github/workflows/master.yml

```
@@ -0,0 +1,485 @@
+
+  sm-linters-e2sm_rc_pre:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd servicemodels/e2sm_rc_pre && golangci-lint run --timeout 5m
+
+  sm-linters-e2sm_rc_pre_go:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd servicemodels/e2sm_rc_pre_go && golangci-lint run --timeout 5m
+
+  sm-linters-e2sm_mho:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd servicemodels/e2sm_mho && golangci-lint run --timeout 5m
+
+  sm-linters-e2sm_rsm:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd servicemodels/e2sm_rsm && golangci-lint run --timeout 5m
+
+  sm-linters-test_sm_aper_go_lib:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd servicemodels/test_sm_aper_go_lib && golangci-lint run --timeout 5m
+
+  sm-linters-protoc-gen-cgo:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd protoc-gen-cgo/ && golangci-lint run --timeout 5m
+
+  sm-linters-protoc-gen-choice:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd protoc-gen-choice/ && golangci-lint run --timeout 5m
+
+  sm-linters-protoc-gen-builder:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          version: v1.54
+      - name: sm-linters
+        run: |
+          cd protoc-gen-builder/ && golangci-lint run --timeout 5m
+        
+  buflint:
+   runs-on: ubuntu-latest
+   steps:
+     - uses: actions/checkout@v4
+     - name: buflint
+       env:
+         BUF_VERSION: 1.0.0
+       run: |
+         sudo mkdir -p /go/src/github.com/onosproject
+         cd /go/src/github.com/onosproject
+         sudo git clone https://github.com/onosproject/onos-e2-sm.git
+         sudo git clone https://github.com/onosproject/onos-lib-go.git
+         cd /go/src/github.com/onosproject/onos-e2-sm
```

I think this should be like follows (which applies to all Actions that are similar to this one)
```suggestion
         cd ..
         sudo git clone https://github.com/onosproject/onos-lib-go.git
         cd onos-e2-sm
```

FYI, you do not need `sudo` in GHAs.